python/Ganga/GPIDev/Adapters/IPrepareApp.py

```diff
@@ -28,7 +28,7 @@ class IPrepareApp(IApplication):
     Base class for all applications which can be placed into a prepared\
     state. 
     """
-    _schema = Schema(Version(0, 0), {'hash': SimpleItem(defvalue=None, typelist=['type(None)', 'str'], hidden=1)})
+    _schema = Schema(Version(0, 0), {'hash': SimpleItem(defvalue=None, typelist=['None', 'str'], hidden=1)})
```

Comment:
Really, I think that the items in `typelist` should not be strings, but rather the types themselves so it would be `typelist=[None, str]` which would avoid having to `eval` these at all. I can;t think of any example where this would be a problem. The code supports this way of doing it already.